### PR TITLE
Add owners file for test images

### PR DESCRIPTION
**Pull request by mkumatag**
**What this PR does / why we need it**:
This PR is for adding OWNERS file for `test/images` and `test/utils/image` folder
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```
NONE
```

### CODE REVIEW

**Review by luxas — APPROVED** [review]
/lgtm

Thanks @mkumatag!

File: test/images/OWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+reviewers:
+  - luxas
+  - mkumatag
+  - ixdy
+approvers:
+  - luxas
+  - ixdy
```
Comment (ixdy) [inline test/images/OWNERS]: this is redundant since I'm already in `test/OWNERS`, but being explicit is also fine.

**Pull request merged**